### ISSUE TITLE
Add DASH_SHIB_ID and default UK Test Fed users for blue UDD Staff

### ISSUE THREAD

**Issue by michaelwebbjisc** [jiscdev/uoj#3]
Amanda to continue testing, needs the UK Test Fed users (beth and duns) added to the current blue UDD data as Staff entities.

She also needs the DASH_SHIB_ID to be filled out for the existing Staff entities in the blue data in order to test student assignments. The DASH_SHIB_ID for entities other than beth and duns can be anything or perhaps match username since they won't be able to login.

(was https://github.com/jiscdev/analytics-udd/issues/154)

**Comment by michaelwebbjisc** [comment]
That's done: Values as follows:

[
  {
    "STAFF_ID": "5000",
    "DASH_SHIB_ID": "craig@test.ukfederation.org.uk",
    "_id": "5823ae162515d52d6d6f5672"
  },
  {
    "STAFF_ID": "5001",
    "DASH_SHIB_ID": "duns@test.ukfederation.org.uk",
    "_id": "5823ae162515d52d6d6f5673"
  },
  {
    "STAFF_ID": "5002",
    "DASH_SHIB_ID": "5002@test.ukfederation.org.uk",
    "_id": "5823ae172515d52d6d6f5674"
  },
  {
    "STAFF_ID": "5003",
    "DASH_SHIB_ID": "5003@test.ukfederation.org.uk",
    "_id": "5823ae172515d52d6d6f5675"
  },
  {
    "STAFF_ID": "5004",
    "DASH_SHIB_ID": "5004@test.ukfederation.org.uk",
    "_id": "5823ae172515d52d6d6f5676"
  },
  {
    "STAFF_ID": "5005",
    "DASH_SHIB_ID": "5005@test.ukfederation.org.uk",
    "_id": "5823ae172515d52d6d6f5677"
  },
  {
    "STAFF_ID": "5006",
    "DASH_SHIB_ID": "5006@test.ukfederation.org.uk",
    "_id": "5823ae182515d52d6d6f5678"
  },
  {
    "STAFF_ID": "5007",
    "DASH_SHIB_ID": "5007@test.ukfederation.org.uk",
    "_id": "5823ae182515d52d6d6f5679"
  },
  {
    "STAFF_ID": "5008",
    "DASH_SHIB_ID": "5008@test.ukfederation.org.uk",
    "_id": "5823ae182515d52d6d6f567a"
  },
  {
    "STAFF_ID": "5009",
    "DASH_SHIB_ID": "5009@test.ukfederation.org.uk",
    "_id": "5823ae182515d52d6d6f567b"
  },
  {
    "STAFF_ID": "5010",
    "DASH_SHIB_ID": "5010@test.ukfederation.org.uk",
    "_id": "5823ae182515d52d6d6f567c"
  },
  {
    "STAFF_ID": "5011",
    "DASH_SHIB_ID": "5011@test.ukfederation.org.uk",
    "_id": "5823ae192515d52d6d6f567d"
  },
  {
    "STAFF_ID": "5012",
    "DASH_SHIB_ID": "5012@test.ukfederation.org.uk",
    "_id": "5823ae192515d52d6d6f567e"
  },
  {
    "STAFF_ID": "5013",
    "DASH_SHIB_ID": "5013@test.ukfederation.org.uk",
    "_id": "5823ae192515d52d6d6f567f"
  },
  {
    "STAFF_ID": "5014",
    "DASH_SHIB_ID": "5014@test.ukfederation.org.uk",
    "_id": "5823ae192515d52d6d6f5680"
  },
  {
    "STAFF_ID": "5015",
    "_id": "5823ae1a2515d52d6d6f5681"
  },
  {
    "STAFF_ID": "5016",
    "_id": "5823ae1a2515d52d6d6f5682"
  },
  {
    "STAFF_ID": "5017",
    "_id": "5823ae1a2515d52d6d6f5683"
  },
  {
    "STAFF_ID": "5018",
    "DASH_SHIB_ID": "5018@test.ukfederation.org.uk",
    "_id": "5823ae1a2515d52d6d6f5684"
  },
  {
    "STAFF_ID": "5019",
    "DASH_SHIB_ID": "5019@test.ukfederation.org.uk",
    "_id": "5823ae1b2515d52d6d6f5685"
  }
]

**Comment by michaelwebbjisc** [comment]
Beth was also needed - did se need to have a particular staff ID?

**Comment by pspaude** [comment]
She used to have 5000, but I'm don't think she needs any particular ID from our end. The only thing dependent on her ID would be her assigned students on TUTOR_STAFF_ID.

**Comment by michaelwebbjisc** [comment]
OK, I'll shift Craig to 5002 and make Beth 5000 again

**Comment by michaelwebbjisc** [comment]
Right, done that - won't close until you've confirmed it's ok this time.

**Comment by amason** [comment]
I can now successfully see and test with Beth. If I run into any other issues, I'll open this ticket back up...but I think we are ok to close this one.
